android/shared/src/commonMain/kotlin/io/rebble/cobble/shared/ui/view/home/watches/WatchesPage.kt

```diff
@@ -0,0 +1,100 @@
+package io.rebble.cobble.shared.ui.view.home.watches
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import io.rebble.cobble.shared.ui.common.RebbleIcons
+import io.rebble.cobble.shared.ui.viewmodel.WatchesListViewModel
+
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun WatchesPage(viewModel: WatchesListViewModel = viewModel{ WatchesListViewModel() }) {
+    val selectedWatch = viewModel.selectedWatch.value
+
+    if (selectedWatch != null) {
+
+        WatchBottomSheetContent(
+                watch = selectedWatch,
+                onToggleConnection = { viewModel.toggleConnection(selectedWatch, true) },
```

Comment:
All the callbacks in `WatchBottomSheetContent` has a parameter `WatchItem` but it is not used here.
I'd suggest either use the parameter or change the signatures of callbacks.
For example:
```suggestion
                onToggleConnection = { viewModel.toggleConnection(it, true) },
```